Resources/Resources.resx

```diff
@@ -876,18 +897,27 @@
   <data name="HttpPOSTCRUDCSharp_description" xml:space="preserve">
     <value>A C# function that adds entities to a Storage Table when it receives an HTTP request</value>
   </data>
+  <data name="HttpPOSTCRUDFSharp_description" xml:space="preserve">
+    <value>An F# function that adds entities to a Storage Table when it receives an HTTP request</value>
+  </data>
   <data name="HttpPOSTCRUDNodeJS_description" xml:space="preserve">
     <value>A Node.js function that adds entities to a Storage Table when it receives an HTTP request</value>
   </data>
   <data name="HttpPUTCRUDC_description" xml:space="preserve">
     <value>A C# function that updates entitiy in a Storage Table when it receives an HTTP request</value>
   </data>
+  <data name="HttpPUTCRUDFSharp_description" xml:space="preserve">
+    <value>An F# function that updates entitiy in a Storage Table when it receives an HTTP request</value>
```

Comment:
Entity is spelled wrong. I see the source you copied from is also wrong. Can you fix that one as well?